### PR TITLE
BZ#1539862 - Created multiselect dialog field type

### PR DESCRIPTION
**Pull request by chalettu**
@miq-bot add_label bug
@miq-bot add_label gaprindashvili/yes

BZ https://bugzilla.redhat.com/show_bug.cgi?id=1539862

### CODE REVIEW

**Comment by chalettu** [comment]
Screenshot of it in action in a dialog in SUI
multiselect

File: src/dialog-user/components/dialog-user/dialogField.html [hunk 1]
```diff
@@ -48,19 +48,38 @@
...
-          <ui-select-choices
-            repeat="fieldValue[0] as fieldValue in vm.dialogField.values | filter: $select.search">
-            <span>{{fieldValue[1]}}</span>
-          </ui-select-choices>
-        </ui-select>
+        <!-- Dropdown field where a single value is expected -->
+        <select pf-select
+                ng-if="!vm.dialogField.options.force_multi_value"
+                ng-model="vm.dialogField.default_value"
+                ng-change="vm.changesHappened(item)"
```
Comment (himdel) [inline src/dialog-user/components/dialog-user/dialogField.html]: Pretty sure `item` is undefined here.. `$item` used to be provided by `ui-select`, but looks like `changesHappened` does `(value ? value[selectedValue] : this.dialogField.default_value)`, so maybe you can drop it and simplify the function a bit?

**Comment by himdel** [comment]
Looks like this should work :+1:

**Comment by chalettu** [comment]
Is this one good to merge?

**Comment by himdel** [comment]
umm

Um.. you tell me ;)

**Comment by JPrause** [comment]
As this is a fix for a blocker issue,...can we get this merged asap. @himdel who can merge?

**Comment by himdel** [comment]
@JPrause understood, in that case, a stray undefined is probably not a problem, merging.

**Comment by himdel** [comment]
@JPrause Is there any need to release a new version of ui-components *now*? (As opposed to monday, in case there are other changes needed here.)

**Comment by chalettu** [comment]
@himdel thank you for merging.  Let me know once we version bump ui-components because I need to update my WIP PR in SUI to reflect the new version.

**Comment by himdel** [comment]
Aah, forgot the other PR was waiting for this too.. in that case, doing the release now..

**Comment by himdel** [comment]
Backported to the `gaprindashvili` branch:

```
commit e32dc5755a19ca6b080e7e957e3e201ba66b98bc
Author: Martin Hradil <himdel@seznam.cz>
Date:   Thu Feb 8 17:34:25 2018 +0100

    Merge pull request #249 from chalettu/multiselect
    
    BZ#1539862 - Created multiselect dialog field type
    
    (cherry picked from commit 2bb47948b307ef2b2374a14bdc7c55db92c01f24)
```

**Comment by himdel** [comment]
Released `@manageiq/ui-components@1.0.10` from the `gaprindashvili` branch (both bower and npm),
and `@manageiq/ui-components@1.1.0` from the `master` branch (npm only).

@miq-bot add_label gaprindashvili/backported
@miq-bot remove_label gaprindashvili/yes

**Pull request merged**